docs/types/operators.rst

```diff
@@ -26,6 +26,19 @@ except for comparison operators where the result is always ``bool``.
 The operators ``**`` (exponentiation), ``<<``  and ``>>`` use the type of the
 left operand for the operation and the result.
 
+Ternary Operator
+----------------
+    The result of the ternary operator (``... ? ... : ...``) is never a number literal, even if all of its operands are number literals.
+    The result type is determined from the types of the two operands in the same way as above, converting to their mobile type first if required.
```

Comment:
This will probably not render properly if you indent the text like this. The header is enough to create a section, you don't have to indent the content in RST or Markdown. The indented text may be interpreted as a code block.